README.md

```diff
@@ -16,15 +16,32 @@ For more details please visit the [documentation](./DOCUMENTATION.md) page.
 
 ## Build it yourself
 
+Download the source code
 ```
-$ git clone https://github.com/tchx84/Flatseal.git
-$ cd Flatseal
-$ flatpak-builder --force-clean --repo=repo build com.github.tchx84.Flatseal.json
-$ flatpak build-bundle repo flatseal.flatpak com.github.tchx84.Flatseal
-$ flatpak install flatseal.flatpak
+git clone https://github.com/tchx84/Flatseal.git
+cd Flatseal
```

Comment:
What I commented before was to keep this in the same code block as the "build with flatpak" instructions.

Reply:
OK. Fixed in 57f0edf